core/src/main/java/hudson/cli/LogoutCommand.java

```diff
@@ -15,7 +15,7 @@
  */
 @Deprecated
 @Extension
-public class LogoutCommand extends CLICommand {
+public class LogoutCommand extends CLICommand implements UnprotectedCLICommand {
```

Comment:
strange question, but if you are trying to log out would you actually need to be Unprotected (otherwise you would be authenticated no?)

Reply:
IIRC, there was a bug before the previous modification for LogoutCommand, you were unable to logout.

Reply:
=> JENKINS-50324